contracts/test/ERC677/erc677-test.ts

```diff
@@ -0,0 +1,129 @@
+import { assert, expect } from 'chai'
```

Comment:
So currently the `package-ts/integration-starkgate` project tests the ERC20 token, which doesn't make much sense to do.

Please merge ERC20tests from there, with these and test the ERC677 & ERC677Mintable tokens. Then also add a test for the `/starkgate/presets/ERC677.cairo` contract.

Notice, these are mostly the same (ERC20 behaviour, + ERC677 behaviour) tests - do not duplicate them, but reuse test code on multiple contracts. For example, check out this implementation of behaviour tests: https://github.com/smartcontractkit/LinkToken/tree/master/test/behavior

----

At that point our `package-ts/integration-starkgate` becomes empty, it's not really testing anything. What we want to test here is a full e2e L1 deposit-> L2 withdraw -> L1 flow. Using our `/starkgate/presets/ERC677.cairo` contract as the token on the other side.

No need to worry about this just jet (testing the bridge), let's first implement ERC677 support and merge this PR.